pca_ica/run_pca.m

```diff
@@ -34,6 +36,12 @@ function run_pca(movie_source, varargin)
                 do_trim = 1;
             case 'medfilt'
                 do_medfilt = 1;
+                if k<length(varargin)

```

Comment:
If `medfilt_halfwidth` is a mandatory input that follows the `'medfilt'` option, then you should write:
```
medfilt_halfwidth = varargin{k+1};
```
(i.e. simply assume that the variable is coming in).